connectors/slack/src/main/java/io/camunda/connector/slack/inbound/model/SlackWebhookProperties.java

```diff
@@ -8,14 +8,39 @@
 
 import com.slack.api.app_backend.SlackSignature;
 import io.camunda.connector.api.inbound.webhook.VerifiableWebhook;
+import io.camunda.connector.generator.dsl.Property;
+import io.camunda.connector.generator.java.annotation.TemplateProperty;
+import io.camunda.connector.generator.java.annotation.TemplateProperty.PropertyType;
+import jakarta.validation.constraints.NotBlank;
 import java.util.Map;
 import java.util.function.Function;
 
 public record SlackWebhookProperties(
-    String context,
-    String slackSigningSecret,
-    Function<Map<String, Object>, VerifiableWebhook.WebhookHttpVerificationResult>
-        verificationExpression) {
+    @TemplateProperty(
+            id = "context",
+            label = "Webhook ID",
+            group = "endpoint",
+            description = "The webhook ID is a part of the URL endpoint")
+        @NotBlank
+        String context,
+    @TemplateProperty(
+            id = "slackSigningSecret",
+            label = "Slack signing secret",
+            group = "endpoint",
+            description =
+                "Slack signing secret. <a href='https://api.slack.com/authentication/verifying-requests-from-slack' target='_blank'>See documentation</a> regarding the Slack signing secret")
+        @NotBlank
+        String slackSigningSecret,
+    @TemplateProperty(
+            id = "verificationExpression",
+            group = "endpoint",
+            type = PropertyType.Hidden,
+            feel = Property.FeelMode.disabled,
```

Comment:
Shouldn't it be `required`? Looks like we're expecting a FEEL expression here

Reply:
The field is hidden, so it doesn't really matter, and if we remove disabled, it sets `feel: optional` which is a bit confusing. In general, Slack has very specific verification expression, so we sort of preventing user to mess with it.